howto/monitor/collect-metrics.md

```diff
@@ -0,0 +1,227 @@
+This example implementation provides a starting point for a monitoring stack that can be used to gather metrics with [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) and [Prometheus](https://prometheus.io/) and access them through [Grafana](https://grafana.com/).
```

Comment:
Should we put some warning that this is only for reference but misses various aspects you will need to consider for a production grade solution (HA, ...)?